src/app/anon/_login.scss

```diff
@@ -1,5 +1,53 @@
+ul
+{
+    list-style-type: none;
+}
+
 .background {
 	width: 100%;
-	height: 100%;
+	height: 1000em;
 	background-color: rgba(0,121,165,1);
-}
\ No newline at end of file
+}
+
+.logInTile {
+  background-color: rgba(164,164,164,.9);
+  border-radius: 5px;
+  margin: 0 15px 30px;
+  padding: 10px 10px 25px;
+  text-align: center;
+  text-decoration: none;
+  position:relative;
+  display: inline-block;
+  overflow: hidden;
+  width: 500px;
+  height: 300px;
+  margin-right: 15px;
+  margin-left: 350px;
+  margin-top: 200px;
```

Comment:
you can combine these all together plus they are duplicated on line 15
did you mean to do this:
this is the same as what you have for all your margins.
```margin: 200px 15px 30px 350px;```
maybe think about doing something like ```margin: 200px auto;```